eventmesh-common/src/main/java/org/apache/eventmesh/common/file/WatchFileManager.java

```diff
@@ -62,10 +60,10 @@ private static void shutdown() {
             return;
         }
 
-        LogUtils.info(log, "[WatchFileManager] start close");
+        log.atInfo().log("[WatchFileManager] start close");
```

Comment:
1.

- Better performance:

>The additional `isDebugEnabled()` check before `log.debug()` is performed in order to avoid invoking the methods referenced in the `message` field, thus improving logging performance. For example:
>
>```java
>log.debug("formatted msg: {}", formatMsg(rawMsg));
>```
>
>Even when the log level is set to `info`, the `formatMsg(rawMsg)` method will still be called and return a result before being passed as an argument to `log.debug()`. This takes time.

- Scenarios where Fluent Logging API is unnecessary:

>Only the scenario mentioned in the Background Knowledge section require the use of Fluent Logging API. For example, in the case of the changes in `Codec` class L224, where no object parameters are passed or existing objects are referenced, there is no additional performance overhead, and there is no need to pre-evaluate log levels. The `log.debug()` usage can be directly applied.

There are many places where the usage of `ifDebugEnabled` is a remnant from the log4j era, which can be replaced with `log.debug()` from slf4j. However, in many cases, removing the `ifDebugEnabled` directly would result in a performance decrease.

——————

有不少地方的`ifDebugEnabled`是log4j时代残留的用法，可以用slf4j的`log.debug()`代替。不过也有很多地方，直接去掉`ifDebugEnabled`的话，是会降低性能的。

2.

That requires obtaining the Fully Qualified Class Name (FQCN) of each class and passing it as a parameter to LogUtils, which is not a good approach for us.

Reply:
> There are many places where the usage of `ifDebugEnabled` is a remnant from the log4j era, which can be replaced with `log.debug()` from slf4j. However, in many cases, removing the `ifDebugEnabled` directly would result in a performance decrease.

To be precise it is a remnant of the Log4j 1.x/JCL era. Since the release of SLF4J in 2005 these guards are not necessary. The Log4j 2.x API (2014) works the same way as SLF4J.

> 2.
> That requires obtaining the Fully Qualified Class Name (FQCN) of each class and passing it as a parameter to LogUtils, which is not a good approach for us.

Probably my explanations were not clear. The FQCN to use is that of the `LogUtils` class, i.e. "org.apache.eventmesh.common.utils.LogUtils". The logging backend will look into the stack for the class that **called** `LogUtils`.

Reply:
>removing the `ifDebugEnabled`

From:

```java
            if (log.isDebugEnabled()) {
                log.debug("Encoder pkg={}", JsonUtils.toJSONString(pkg));
            }
```

To:

```java
           log.debug("Encoder pkg={}", JsonUtils.toJSONString(pkg));
```

Reply:
这里的性能会更好，应该还是部分特殊的情况（适用于能够懒加载的情况），既然LogUtils能解决日志代码行显示的问题，我觉得也可以采用LogUtils统一封装一下，这样在编码风格上也能有一定的统一。